src/components/common/WordSelectorWithFourWords/WordSelectorWithFourWords.jsx

```diff
@@ -0,0 +1,39 @@
+import wordObjects from '../../../data/wordObjects';  // Importing an array with data
+import { getRandomWords } from '../../../utils/getRandomWords';  // Utility for getting random words
+
+const WordSelectorWithFourWords = ({
+  mainWord,
+  setRandomWords,
+  setShowAdditionalWords,
+}) => {
+
+  // Function for displaying additional words
+  const showFourWords = () => {
+    if (mainWord) {
+      // Filter words excluding the main word
```

Comment:
Please remove such comments across the entire project. They unnecessarily duplicate function names and hinder your ability to learn, as they encourage you to rely on simple human language rather than reading the code